src/tests/functional/plugin/myriad/ngraph/transformations/convert_extract_image_patches_to_reorg_yolo_vpu.cpp

```diff
@@ -130,8 +130,7 @@ INSTANTIATE_TEST_SUITE_P(smoke_NGraph, DoNotConvertEIPToReorgYoloOnNot4DInput, t
         testing::Values(ngraph::PartialShape{1, 1, 64, 500, 500},
                         ngraph::PartialShape{64, 500, 500},
                         ngraph::PartialShape{500, 500},
-                        ngraph::PartialShape{500},
-                        ngraph::PartialShape::dynamic()),
```

Comment:
@v-Golubev Hi, after I fix the `extract_image_patches`, the input with `PartialShape::dynamic()` doesn't throw any exception, therefore I remove this situation in the myriad tests.